plugin/output/postgres/postgres.go

```diff
@@ -65,6 +66,7 @@ type Plugin struct {
 	ctx        context.Context
 	cancelFunc context.CancelFunc
 
+	backoff      backoff.BackOff
```

Comment:
do we need need this only to propagate calcellation to send() ?

how about taking it from elsewere?

for example from https://github.com/ozontech/file.d/blob/058ba19c99fcffdd6c6fef427adab5bc8e788bec/pipeline/batch.go#L62-L62

also, but it goes out of scope of this PR of course, setted way to use context is via params. better way to fix this would be imo passing it here http://github.com/ozontech/file.d/blob/master/pipeline/pipeline.go#L53-L53 but it is too much of a change right now

Reply:
We use it both for canellation and for backoff timer stretch during retry.

Reply:
Cancellation where? I cant find it

Reply:
Ctx&retry cancellation sets here: https://github.com/ozontech/file.d/blob/b863c884c770d4c37713997b83d6c30f989ed715/plugin/output/postgres/postgres.go#L249-L250

Reply:
yes that is what I am saying you create context and pass it in struct only to use later in backoff timer 
https://github.com/ozontech/file.d/blob/b863c884c770d4c37713997b83d6c30f989ed715/plugin/output/postgres/postgres.go#L350

you do not need it if you just check shouldStop()

cancellation in your commit is still done with another ctx
https://github.com/ozontech/file.d/blob/b863c884c770d4c37713997b83d6c30f989ed715/plugin/output/postgres/postgres.go#L339

Reply:
Oh yes I can see now.
Well, I still think that two new contextes, passing it through struct and a new dependency is too much for a task that can be done just by a  for loop

Reply:
We can do everything on programming with `for` loops. But it's not worth it.